app/lib/search-validation.js

```diff
@@ -3,7 +3,7 @@ const validStatus = ['agreed', 'applied', 'withdrawn', 'data inputted', 'claimed
 const sbiRegEx = /^[\0-9]{9}$/i
 
 module.exports = (searchText) => {
-  let searchType
+  let searchType = 'organisation'
```

Comment:
Nit - Could maybe have this as the default case in the switch instead of defaulting to organisation?